advanced_functionality/pytorch_deploy_large_GPT_model/GPT-J-6B-model-parallel-inference-DJL.ipynb

```diff
@@ -0,0 +1,383 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.    aws s3 cp gpt-j.tar.gz  #S3 path such as s3://djl-big-model/deepspeed/
</pre><p>Please use the sagemaker default S3 bucket and a demo path (like <code>DEMO-djl-big-model/</code>), rather than requiring the user to provide their own S3 path manually.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3540/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
@billdoors completed this